pgbouncer/templates/service.yaml

```diff
@@ -2,23 +2,20 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Values.service.name }}
+  name: {{ default (include "pgbouncer.fullname" .) .Values.service.name }}
   labels:
-    {{ template "pgbouncer.appLabel" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    {{- include "pgbouncer.labels" . | nindent 4 }}
   annotations:
     {{- range $key, $value := .Values.service.annotations }}
       {{ $key }}: {{ $value | quote }}
     {{- end }}
 spec:
   type: {{ .Values.service.type }}
   selector:
-    {{ template "pgbouncer.appLabel" . }}
+    {{- include "pgbouncer.selectorLabels" . | nindent 6 }}
   ports:
-    - port: {{ .Values.servicePort }}
+    - port: {{ .Values.service.port }}
```

Comment:
see comment below about changing servicePort.  We need to keep this as-is for backward compatibility.

Reply:
This is done, it is using servicePort now.